charts/karmada/templates/kube-controller-manager.yaml

```diff
@@ -54,6 +54,7 @@ spec:
             - --authentication-kubeconfig=/etc/kubeconfig
             - --authorization-kubeconfig=/etc/kubeconfig
             - --bind-address=0.0.0.0
+            - --secure-port=10257
```

Comment:
Why does the YAML file in `artifacts/deploy/kube-controller-manager.yaml` contain this parameter?

Reply:
I not use `artifacts/deploy/kube-controller-manager.yaml` deploy kube-conftroller-manager,  I don't know Whether it will have the same problem

Reply:
Which version of  `kube-controller-manager` do you use?

Reply:
Both `v1.22.10` and `v1.24.2` have this problem, I've used them all